client/src/App.js

```diff
@@ -1,7 +1,28 @@
 import React from 'react';
+import CardComponent from '../src/components/Card';
+import styled from '@emotion/styled';
+import { ThemeProvider } from 'emotion-theming';
+import GlobalStyles from './GlobalStyles.js';
+import main from './themes/defaultTheme.js';
+import HeaderNav from './components/Header.js';
 
-function App() {
-  return <div>Hallo Welt</div>;
-}
+const CardWrapper = styled.div`
+  margin-top: 110px;
+`;
 
-export default App;
+export default function App() {
+  return (
+    <>
+      <ThemeProvider theme={main}>
+        <GlobalStyles />
+        <HeaderNav />
+        <CardWrapper>
+          <CardComponent></CardComponent>
```

Comment:
I think it is more readable if the name of the function in the component and the component file have the same name :)
You have a "Card.js" so the name of the function in this component should be "export default function Card()..."